cloud-sql/postgres/activerecord/bin/run_tests

```diff
@@ -15,4 +15,4 @@ bundle exec rspec --format documentation --format RspecJunitFormatter --out spon
 
 # Run the tests again with custom environment
 # variables that force a Unix connection
-DB_HOST=$SOCKET_PATH; bundle exec rspec --format documentation --format RspecJunitFormatter --out sponge_log.xml | tee sponge_log.log
\ No newline at end of file
+INSTANCE_UNIX_SOCKET=$SOCKET_PATH; INSTANCE_HOST=$SOCKET_PATH; bundle exec rspec --format documentation --format RspecJunitFormatter --out sponge_log.xml | tee sponge_log.log
```

Comment:
Could we break this across a couple lines? It's hard to read as is.

```
INSTANCE_UNIX_SOCKET=$SOCKET_PATH \
  INSTANCE_HOST=$SOCKET_PATH \
  bundle exec rspec --format documentation \
  --format RspecJunitFormatter --out sponge_log.xml | tee sponge_log.lo
```

Reply:
Done.